src/app/forms/types/tooltip.type.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '@angular/core';
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  selector: 'tooltip-code',
+  styleUrls: [],
+  styles: [
+    `
+.label{
+  font-size: 12px;
+  font-weight: 700;
+  margin-bottom: 0.5rem;
+  color: var(--secondaryTextColor);
+}
+
+.tooltip-text {
+  display: none;
+  background-color: #000;
+  color: #fff;
+  text-align: left;
+  border-radius: 6px;
+  position: absolute;
+  padding: 3px;
+  margin-left: 10px;
+}
+
+.tooltip-wrapper:hover + .tooltip-text {
+  display: inline-block;
+  position: absolute;
+}
+
+.tooltip-wrapper:hover + .tooltip-text::before {
+  content: "";
+  position: absolute;
+  border-top: 5px solid transparent;
+  border-bottom: 5px solid transparent;
+  border-right: 5px solid #000;
+  left: -5px;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+    `
+  ],
+  template: `
+    <div >
+        <label class="tooltip-wrapper label">{{ to.label }}
+          <span *ngIf="to.required" class="red mr-1">*</span>
+          <span><i class="fa fa-info-circle" style="color:#000;"></i></span>
+        </label>
+        
+        <span class="tooltip-text">{{ to.tooltip }}</span>
+        <input type="text" class="form-control" [formControl]="formControl" [formlyAttributes]="field" [title]="to.tooltip">
+        <small class="form-text text-muted">{{ to.description }}</small>
+        
+        
+    </div>  
+`
+})
+export class TooltipWrapper extends FieldType {
+  tooltipWrapper: any;
+  tooltipText: any;
+  get tooltipContent(): string {
```

Comment:
@suvarnakale is it requred?

Reply:
I added that for mouse hover and mouseOut but it is also managable by css, added later. I will remove it.

Reply:
so are you using this rmethod right now or not?

Reply:
Not using the get method.. will remove it.. previously I was confused with varible declarations and function.